includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -116,7 +118,7 @@ public function setup_wizard() {
 		) );
 
 		if ( ! empty( $_POST['save_step'] ) && isset( $this->steps[ $this->step ]['handler'] ) ) {
-			call_user_func( $this->steps[ $this->step ]['handler'] );
+			call_user_func( $this->steps[ $this->step ]['handler'], $this->get_next_step_link() );
```

Comment:
@claudiosanches I know this was your suggestion - I don't feel we should pass around the next step link like this. There may be instances for example where you also need the *previous* step link? Or to know more about the current step?

Since these functions are callbacks, and they should in theory know what step they are for (not the numeric step, but the step's key e.g. `introduction`), why not have a method which can lookup the next step from the current key?

e.g, in the callback use something like:

`
$this->get_next_step_link( 'introduction' ); // introduction is the current step ID
`

?